src/main/java/reposense/RepoSense.java

```diff
@@ -62,7 +67,11 @@ public static void main(String[] args) {
                     cliArguments.getSinceDate().orElse(null),
                     cliArguments.getUntilDate().orElse(null));
 
-            FileUtil.zip(cliArguments.getOutputFilePath().toAbsolutePath(), ".json");
+            HashSet<Path> relevantFolders = getRelativeFolders(cliArguments.getOutputFilePath(), configs);
+            HashSet<Path> relevantFiles = new HashSet<>();
+            relevantFiles.add(Paths.get(SummaryReportJson.SUMMARY_JSON_FILE_NAME));
+            FileUtil.zipRelativeFiles(relevantFolders, relevantFiles, cliArguments.getOutputFilePath().toAbsolutePath(),
```

Comment:
Similar to `zip()`, you can overload this method so you do not need to call `cliArguments.getOutputFilePath().toAbsolutePath()` twice.